assets/sass/elements/badge.scss

```diff
@@ -20,6 +20,15 @@
   &:empty {
     display: none;
   }
+
+  background: var(--wider-colour, transparent);
+
+  &:is(:hover,.hover) {
+    background: var(--wider-colour-hover, transparent);
+  }
+  &:is(:active,.active) {
```

Comment:
Is it worth adding a visited style?

Reply:
We haven't added any other visited styles into the system. I have always seen little benefit in them with the exception of inline links.

Reply:
That's fair enough 👍 All looks good otherwise.